docs/content/latest/integrations/jdbc-driver.md

```diff
@@ -0,0 +1,278 @@
+
+### Obtain the driver from Maven
+
+To get the driver and HikariPool from Maven, add the following lines to your Maven project:
+
+```xml
+<dependency>
+  <groupId>com.yugabyte</groupId>
+  <artifactId>jdbc-yugabytedb</artifactId>
+  <version>42.3.0</version>
+</dependency>
+
+<!-- https://mvnrepository.com/artifact/com.zaxxer/HikariCP -->
+<dependency>
+  <groupId>com.zaxxer</groupId>
+  <artifactId>HikariCP</artifactId>
+  <version>4.0.3</version>
+</dependency>
+```
+
+### Build the driver
+
+To build the driver locally, follow this procedure:
+
+1. For the build environment, [gpgsuite](https://gpgtools.org/) needs to be present on the machine where the build is performed. Install GPG and create a key.
+
+2. Clone the following repository:
+
+   ```sh
+   git clone https://github.com/yugabyte/pgjdbc.git && cd pgjdbc
+   ```
+
+3. Checkout the `yugabyte` branch.
+   
+   ```sh
+   git checkout yugabyte
+   ```
+
+4. Build and install into your local Maven folder.
+   
+   ```sh
+   ./gradlew publishToMavenLocal -x test -x checkstyleMain
+   ```
+
+5. Add the following lines to your Maven project:
+
+   ```xml
+   <dependency>
+     <groupId>com.yugabyte</groupId>
+     <artifactId>jdbc-yugabytedb</artifactId>
+     <version>42.3.0</version>
+   </dependency> 
+   ```
+
+{{< note title="Note">}}
+
+The driver requires YugabyteDB version 2.7.2.0 or higher, and Java 8 or above.
+
+{{< /note >}}
+
+### Load balancing connection properties
+
+The following connection properties need to be added to enable load balancing:
+
+- load-balance - enable cluster-aware load balancing by setting this property to `true`; disabled by default.
+- topology-keys - provide comma-separated geo-location values to enable topology-aware load balancing. Geo-locations can be provided as `cloud:region:zone`.
+
+## Use the driver
+
+To use the Yugabyte JDBC Driver, do the following:
+
+- Pass new connection properties for load balancing in the connection URL or properties pool.
+
+  To enable uniform load balancing across all servers, you set the `load-balance` property to `true` in the URL, as per the following example:
+
+  ```java
+  String yburl = "jdbc:yugabytedb://127.0.0.1:5433/yugabyte?user=yugabyte&password=yugabyte&load-balance=true";
+  DriverManager.getConnection(yburl);
+  ```
+
+  To specify topology keys, you set the `topology-keys` property to comma separated values, as per the following example:
+
+  ```java
+  String yburl = "jdbc:yugabytdb://127.0.0.1:5433/yugabyte?user=yugabyte&password=yugabyte&load-balance=true&topology-keys=region1:zone1,region1.zone2";
+  DriverManager.getConnection(yburl);
+  ```
+
+- Configure `YBClusterAwareDataSource` for uniform load balancing and then use it to create a connection, as per the following example:
+
+  ```java
+  String jdbcUrl = "jdbc:yugabytedb://127.0.0.1:5433/yugabyte";
+  YBClusterAwareDataSource ds = new YBClusterAwareDataSource();
+  ds.setUrl(jdbcUrl);
+  ds.setLoadBalance("true");
```

Comment:
Please remove this line. The YBClusterAwareDataSource has 'load-balance' on by default.